packages/sku/lib/parseArgs.js

```diff
@@ -1,75 +1,58 @@
-const commandLineArgs = require('command-line-args');
-
-const optionDefinitions = [
-  {
-    name: 'script',
-    defaultOption: true,
-  },
-  {
-    name: 'env',
-    alias: 'e',
-    type: String,
-    defaultValue: 'production',
-  },
-  {
-    name: 'tenant',
-    alias: 't',
-    type: String,
-    defaultValue: '',
-  },
-  {
-    name: 'build',
-    alias: 'b',
-    type: String,
-  },
-  {
-    name: 'config',
-    alias: 'c',
-    type: String,
-  },
-  {
-    name: 'debug',
-    alias: 'D',
-    type: Boolean,
-  },
-  {
-    name: 'environment',
-    type: String,
-  },
-  {
-    name: 'packageManager',
-    type: String,
-  },
-  {
-    name: 'port',
-    type: Number,
-  },
-  {
-    name: 'site',
-    type: String,
-  },
-  {
-    name: 'stats',
-    type: String,
-  },
-];
+const minimist = require('minimist');
 
 /**
  * Supports parsing args that look like:
  * [/path/to/node/node, /path/to/sku, scriptName, arg1, arg2]
  *
  * @param {string[]} args - should look like process.argv
  */
-module.exports = (args) => {
-  const options = commandLineArgs(optionDefinitions, {
+module.exports = (processArgv) => {
+  /**
+   * @type {string[]}
+   */
+  const unknown = [];
+
+  // We are tracking unknown arguments ourselves, so we ignore `minimist`'s unknown property `_`
+  const { _, ...options } = minimist(
     // The first 2 items in process.argv are /path/to/node and /path/to/sku.
-    // We need the first arg we give to command-line-args to be the script name.
-    argv: args.slice(2),
-    stopAtFirstUnknown: false,
-    partial: true,
-  });
+    // We need the first arg we give to minimist to be the script name.
+    processArgv.slice(2),
+    {
```

Comment:
`port` is not specified as numeric arguments are returned as `number`s by default.